requirements.txt

```
@@ -1,4 +1,4 @@
-fairseq2>=0.4.0
+fairseq2>=0.5.0
```

I wonder if we can get rid off requirements.txt to use only pyproject as a single reference
```suggestion
fairseq2>=0.5.2
```

---

We are using them only in the Github CI, so why not.